### DISCUSSION TITLE
[已解决]第一步启动starcoin节点过不去，请求老师指点指点

### DISCUSSION THREAD

**Discussion by zhaoge5859** [NonceGeek/Move-Camp#155] [Q&A] [answered]
小白一个，刚接触2天，我本地电脑安装了ubuntu，按照starcoin cookbook节点安装教程做的，一直报错，查了很多资料，没解决，我考虑可能是本地电脑的事，然后昨天晚上，我换了一台服务器，安装的同版本ubuntu，报错还是如下图所示，求老师们指点，卡在节点启动过不去

![WechatIMG6](https://user-images.githubusercontent.com/116735858/198780178-16e79d84-fca8-4d9d-8deb-c8f5ffee5e9e.jpeg)
![WechatIMG7](https://user-images.githubusercontent.com/116735858/198780213-ee14b40c-d3fe-40ae-81ea-1272dab3f508.jpeg)

**Comment by MagicGordon** [comment, marked as answer]
根据提示来就好，执行sudo apt-get install  libssl-dev

**Reply by zhaoge5859** [reply]
试过了 不行

**Reply by zhaoge5859** [reply]
不知道倒底是哪里出问题了 ，刚才把服务器重新装了一下系统 ，还是这个提示

**Reply by zhaoge5859** [reply]
用docker安装解决了

**Reply by MagicGordon** [reply]
嗯嗯，用官方docker镜像最省事，我用Ubuntu docker试了一下源码安装没遇到问题。  上面编译报错感觉./scripts/dev_setup.sh脚本执行就应该没成功，我看脚本里有安装libssl-dev的步骤。
